js/server/modules/@arangodb/foxx/queues/manager.js

```diff
@@ -64,9 +71,13 @@ var runInDatabase = function () {
           }
 
           jobs.forEach(function (job) {
-            db._jobs.update(job, {
+            const update = {
               status: 'progress'
-            });
+            };
+            if (isCluster) {
+              update.startedBy = coordinatorId;
```

Comment:
Is this used for anything?

Reply:
No. But @mchacki and me agreed that it would be useful information to have if only for debugging, and might be useful later for additional functionality.